drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -204,7 +183,7 @@ def do_attempt(self, attempt, suffix=''):
                 # If this item has no input value set, we are done with it.
                 is_correct = True
                 feedback = item['feedback']['correct']
-            state = {'top': attempt['top'], 'left': attempt['left']}
+            state = {'top': attempt['top'], 'left': attempt['left'], 'position': 'absolute'}
```

Comment:
@mtyaka I see - it might be worth making it obvious that it's a flag, or at least using a different name/value than what is used in CSS, as it looks like it's going to be directly translated to a CSS property currently.